crates/re_types/definitions/rerun/blueprint/components/visible_time_range.fbs

```diff
@@ -0,0 +1,30 @@
+include "arrow/attributes.fbs";
+include "python/attributes.fbs";
+include "rust/attributes.fbs";
+
+include "rerun/datatypes.fbs";
+include "rerun/attributes.fbs";
+
+namespace rerun.blueprint.components;
+
+// ---
+
+/// The range of values on a given timeline that will be included in a view's query.
+///
+/// Whenever no visual time range applies, queries are done with "latest at" semantics.
+/// This means that the view will, starting from the time cursor position,
+/// query the latest data available for each component type.
+///
+/// The default visual time range depends on the type of view this property applies to:
+/// - For time series views, the default is to show the entire timeline.
+/// - For any other view, the default is to apply latest-at semantics.
```

Comment:
This seems like it belongs on the archetype. The default is relevant when the `ranges` in the archetype does not have a value for the given timeline.